js/tiles.js

```diff
@@ -292,4 +292,12 @@ const door = {
       this.ground.show();
     },
   }),
+  outdoor: Object.assign({}, OpenDoors, {
+    canEnterFromTheRight() { return false; },
+    canLeaveToTheRight() { return false; },
+    createImages: function () {
+      this.wallRight = this.createImage("tiles/rooms/wall/outwall.svg");
+      this.ground = this.createImage("tiles/rooms/floor/outdoor.svg");
+    },
+  });
```

Comment:
I think, the ; is too much. It could be a , instead.

Reply:
I will merge it as soon as I can play it.

Reply:
oh yes